examples/oneapi/cpp/source/decision_forest/df_reg_dense_batch.cpp

```diff
@@ -18,45 +18,47 @@
 #include "oneapi/dal/algo/decision_forest.hpp"
 #include "oneapi/dal/io/csv.hpp"
 
-using namespace oneapi;
+namespace onedal = oneapi::dal;
 namespace df = oneapi::dal::decision_forest;
 
-
 int main(int argc, char const *argv[]) {
-  const std::string train_data_file_name  = get_data_path("df_regression_train_data.csv");
-  const std::string train_label_file_name = get_data_path("df_regression_train_label.csv");
-  const std::string test_data_file_name   = get_data_path("df_regression_test_data.csv");
-  const std::string test_label_file_name  = get_data_path("df_regression_test_label.csv");
+    const std::string train_data_file_name = get_data_path("df_regression_train_data.csv");
+    const std::string train_label_file_name = get_data_path("df_regression_train_label.csv");
+    const std::string test_data_file_name = get_data_path("df_regression_test_data.csv");
+    const std::string test_label_file_name = get_data_path("df_regression_test_label.csv");
 
-  const auto x_train = dal::read<dal::table>(dal::csv::data_source{train_data_file_name});
-  const auto y_train = dal::read<dal::table>(dal::csv::data_source{train_label_file_name});
+    const auto x_train =
+        onedal::read<onedal::table>(onedal::csv::data_source{ train_data_file_name });
```

Comment:
Can we make it shorter, or compiler error occurs?
```suggestion
    const auto x_train =
        onedal::read<onedal::table>({ train_data_file_name });
```

Reply:
@michael-smirnov, not sure it's good idea, `read` is invariant for multiple data sources. This may demonstrate wrong usage scenario in case where multiple data sources can be constructed from a string. Currently it works, because we have only one data source. When a new data source is added, for example `hdf5` that also accepts path to file, the suggested example leads to the compilation error due to ambiguity between `csv::data_source` and `hdf5::data_source` constructors.

Reply:
Agree